hydro/Routing/Reservoirs/RFC_Forecasts/CMakeLists.txt

```diff
@@ -0,0 +1,8 @@
+add_library(hydro_routing_reservoirs_rfc STATIC
```

Comment:
Also needed netCDF-Fortran "linking"

Reply:
Included NetCDF directories. Thanks for pointing these linking issues out! For some reason it worked perferctly on Derecho so didn't know this was happening. Was able to reproduced with a local OSX build so these all should be working correctly now.

Reply:
I think Derecho has a tendency to fill up your environment with a ton of stuff, often making sure software _will_ work but hides any issues that might happen in other environments.